app/controllers/admin/resources_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Admin
+  class ResourcesController < ApplicationController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :resource, only: [:show, :edit, :update, :destroy]
```

Comment:
who should be able to create, modify, detroy and see resources? Currently only admins would be allowed right? I would say that organizers should be able to manage all this too. And users signed in with info desk role and volunteers coordinator role should be able to see and maybe update them too.